examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -0,0 +1,1546 @@
+{
```

Comment:
<p>"The NVTabular Op <code>GroupBy</code> enables the transformation" rephrase as "The NVTabular <code>GroupBy</code> op enables the transformation."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>